wiki/People/Completionists/en.md

```diff
@@ -0,0 +1,27 @@
+# Completionists
+
+**Completionists** are players who have completed every [Ranked](/wiki/Beatmap/Category#ranked) [beatmap](/wiki/Beatmap) of a given [game mode](/wiki/Game_mode). Because the amount of Ranked beatmaps increases every day, this feat becomes progressively more difficult to achieve.
+
+As of August 2023, this achievement is recognised through a [profile badge](/wiki/Community/Profile_badge)
+
+![osu!](img/osu.png "osu! completionist badge") ![osu!taiko](img/taiko.png "osu!taiko completionist badge") ![osu!catch](img/catch.png "osu!catch completionist badge") ![osu!mania](img/mania.png "osu!mania completionist badge")
+
+## Completion criteria
+
+In order to be considered a completionist, a player must play every existing Ranked beatmap of a given game mode with no [difficulty reduction mods](/wiki/Gameplay/Game_modifier#difficulty-reduction). This does not include converted beatmaps for [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania). For osu!mania, only 4K and 7K beatmaps are considered for completion.
+
+## Confirmed completionists
+
+| Player | Game mode | Date of completion | Total played difficulties |
+| :-- | :-- | :-- | :-- |
+| ::{ flag=US }:: [xasuma](https://osu.ppy.sh/users/3172980) | ![](/wiki/shared/mode/osu.png) | 2019-04-27 | 60,845 |
+| ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352) | ![](/wiki/shared/mode/taiko.png) | 2019-11-03 | 8,841 |
+| ::{ flag=PL }:: [PookieBear](https://osu.ppy.sh/users/7635621) | ![](/wiki/shared/mode/osu.png) | 2021-08-11 | 82,298 |
+| ::{ flag=US }:: [EEEEEEEEEEEEEEE](https://osu.ppy.sh/users/2927048) | ![](/wiki/shared/mode/osu.png) | 2023-3-15 | 97,668 |
+| ::{ flag=MY }:: [\[Zeth\]](https://osu.ppy.sh/users/9912966) | ![](/wiki/shared/mode/osu.png) | 2023-08-14 | 22,170 |
```

Comment:
```suggestion
| ::{ flag=MY }:: [\[Zeth\]](https://osu.ppy.sh/users/9912966) | ![](/wiki/shared/mode/taiko.png) | 2023-08-14 | 22,170 |
```
wrong mode

Reply:
also i believe missing some people?

AutoLs (catch, all fcs)
Araxcrow (mania, completion)

may need verifying

Reply:
i've intentionally only included people that have been properly verified now, others can be added later if they somehow manage to get verified